apps/geoportal/src/app/store/slices/mapping.ts

```diff
@@ -220,73 +236,48 @@ export const {
   setShowLocatorButton,
   setShowMeasurementButton,
   setShowHamburgerMenu,
+  
   setFocusMode,
+  setClickFromInfoView,
   setStartDrawing,
+  
   toggleUseInFeatureInfo,
-  setClickFromInfoView,
 } = slice.actions;
 
-export const getLayers = (state: RootState) => {
-  return state.mapping.layers;
-};
-
-export const getSavedLayerConfigs = (state: RootState) => {
-  return state.mapping.savedLayerConfigs;
-};
-
-export const getSelectedLayerIndex = (state: RootState) => {
-  return state.mapping.selectedLayerIndex;
-};
-
-export const getSelectedMapLayer = (state: RootState) => {
-  return state.mapping.selectedMapLayer;
-};
-
-export const getBackgroundLayer = (state: RootState) => {
-  return state.mapping.backgroundLayer;
-};
-
-export const getLayerState = (state: RootState): LayerState => {
-  return {
```

Comment:
@d4v3000 this creates a new object every time the selector is called. And will trigger rerenders, since the objects are different